Platforms/QemuSbsaPkg/PlatformBuild.py

```diff
@@ -6,15 +6,8 @@
 ##
 import os

```

Comment:
could you please perform an isort on this? There is a vscode plugin to do it for you.

Reply:
Same on the QemuQ35Pkg side too if you don't mind.

Reply:
sorted the imports for both packages.